frontend/src/components/explorer/MapViewer.vue

```diff
@@ -163,7 +163,7 @@ export default {
       if (this.avail2D) {
         return `Switch to ${this.dimensionalState(!this.showing2D)}`;
       }
-      return 'This model only has 3D maps available';
+      return 'This model has only 3D maps available';
```

Comment:
Why changing the MapViewer in this PR?

Reply:
It was an unresolved comment from [a previous PR](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/786#discussion_r766683723).

Reply:
Also think this is unrelated to this PR, Would create another PR for this

Reply:
A PR without an issue then? Is that acceptable?

Reply:
For typo kind of things, i.e. changing one or two words in one file, I believe so. Things that does not require an extensive review. If a change affects several files/actual code functionality, an new issue should be created IMO. But apparently something that is not established so we should discuss it further.

What we have here is a PR related to an issue with changes unrelated to that issue, I'm not fan of that

Reply:
I'm also not a fan of the amount of clicks and typing required in a PR template for what could essentially be a typo vs a single, individual commit with clear explanation.

This commit has been moved to a separate PR https://github.com/MetabolicAtlas/MetabolicAtlas/pull/802

Reply:
From what I remember, what we have established before was to check with the team if an un-established action is intended.

In this case, @mihai-sysbio did ask about this in another PR but no one responded so I feel like the rest of the team could have responded better there. Personally I feel like it was not a big difference either way but I could have written that instead of ignoring it. I'm also ok with including tiny changes like this in any unrelated PR (such as this one), but this is not what we have previously agreed on and it would have been nice to discuss with the team. Yes it is tedious to have a discussion about such a tiny fix, but if we can establish a new norm from that discussion it would help us avoid these situations in the future.